ReactNativeClient/lib/path-utils.js

```diff
@@ -100,9 +100,18 @@ function friendlySafeFilename(e, maxLength = null) {
 	return output.substr(0, maxLength);
 }
 
-function toFileProtocolPath(path) {
-	const output = path.replace(/\\/g, "/");
-	return 'file://' + escape(output);
+function toFileProtocolPath(filePathEncode) {
```

Comment:
Platform should be passed a parameter so that the tests can be done whatever the platform is. Platform can be optional and, if not provided, can be set to process.platform as in your current code.